object_storage_api/schemas/attachment.py

```diff
@@ -21,27 +19,27 @@ class AttachmentPostSchema(BaseModel):
 
 
 class AttachmentPostUploadInfoSchema(BaseModel):
-    """
-    Schema model for the information required to upload a file.
-    """
+    """Schema model for the information required to upload a file."""
 
     url: HttpUrl = Field(description="Pre-signed upload URL to upload the attachment file to")
     fields: dict = Field(description="Form fields required for submitting the attachment file upload request")
 
 
 class AttachmentMetadataSchema(AttachmentPostSchema):
-    """
-    Schema model for an attachment's metadata.
-    """
+    """Schema model for an attachment's metadata."""
 
     id: str = Field(description="ID of the attachment")
 
 
 class AttachmentPostResponseSchema(CreatedModifiedSchemaMixin, AttachmentMetadataSchema):
-    """
-    Schema model for the response to an attachment creation request.
-    """
+    """Schema model for the response to an attachment creation request."""
 
     upload_info: AttachmentPostUploadInfoSchema = Field(
         description="Information required to upload the attachment file"
     )
+
+
+class AttachmentSchema(AttachmentMetadataSchema):
+    """Schema model for an attachment get request response."""
+
+    url: HttpUrl = Field(description="Presigned get URL to get the attachment file")
```

Comment:
```suggestion
    dowload_url: HttpUrl = Field(description="Presigned get URL to download the attachment file")
```
To match how we are now distinguishing URLs in #94.